onnxscript/tests/function_libs/torch_lib/ops_test.py

```diff
@@ -212,8 +212,12 @@ def run_test_output_match(
                     torch_output = torch.view_as_real(torch_output)
 
                 reference_torch_outputs, _ = pytree.tree_flatten(torch_output)
-                if op.name.startswith("split") or op.name.startswith("chunk"):
-                    # Hack for handling split and chunk
+                if (
+                    op.name.startswith("split")
+                    or op.name.startswith("chunk")
+                    or op.name.startswith("unbind")
```

Comment:
Growing

Reply:
I suspect we can know look at the function signature for this?

Reply:
Oh yup. We can leverage on Opschema and get output dtype.

Reply:
I had a try yesterday but failed to find a clear solution.